styleguide/source/_patterns/01-molecules/09-search-results/search-results-best-match-event.twig

```diff
@@ -0,0 +1,29 @@
+{#
+ # @file search-results-best-match-event.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<div class="layout layout_one_up">
+  {% include 'atoms-h6' with { 'class' : 'search-result_label', 'content' : 'Best Match' } %}
+  <div class="search-result">
+    <div class="search-result_left">
+      <div class="search-result_left-inner">
+        {% include 'atoms-h4' with { 'class' : 'search-result_title', 'content' : 'Lorem Ipsum Dolor Amet Consectetur Adipiscing Elit of Laoreet Pharetra' } %}
+        {% include 'atoms-paragraph' with { 'class' : 'search-result_description-event', 'content' : 'Sed massa augue, sollic itudiny vel metus a, posuere erat. In vulputate feugiat neque, sed aliquam juh tasga hugit.' } %}
+
+        {% include 'atoms-paragraph' with { 'class' : 'search-result_left-event-details', 'content' : 'June 11–15, 2016'} %}
+        {% include 'atoms-paragraph' with { 'class' : 'search-result_left-event-details', 'content' : 'Hyatt Regency Chicago'} %}
+        {% include 'atoms-paragraph' with { 'class' : 'search-result_left-event-details', 'content' : 'Chicago, IL'} %}
```

Comment:
I also wonder if we can use flex-box order to keep this item the same and not have to repeat it at mobile. If we structure the markup up title desc cta event details, and then at mobile, using order, put the event details beneath the title, I think it could work without having to repeat any code.